tox.ini

```diff
@@ -96,3 +96,12 @@ deps=
     toml
 commands=
     bandit -c bandit.yaml -r .
+
+[testenv:warnings]
```

Comment:
Do you need a special job for this? Why not set this global for pytest in all the jobs?

Reply:
We don't like new warnings killing all of our CI. This one services as a friendly reminder to fix warnings without causing unrelated failures for normal development and testing.

Reply:
But depending on how the matrix is set up, some warning might appear in one job but not another.